README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
+
+My *hobbies* include playing sports such as basketball, baseball, and volleyball. I enjoy the **outdoors** as much as I enjoy spending a lazy day **indoors**. I care deeply about the social, economic, and civil challenges facing our democracy today and I actively donate my time, energy, and resources to many causes I find of pressing importance. I *source* my news at times from [link](https://www.npr.org/) [link](https://www.washingtonpost.com/) or [link](https://thehill.com/) 
```

Comment:
I like that you included multiple links! Maybe you could make the text the names of the websites just so that it's clear where the links go? But either way good job!

Reply:
I liked how you're experimenting with different fonts and bolded "outdoors" and "indoors." I also agree with Emma I think it would be more clear if you replace the "link" with actual website names, so we have a more clear idea about specifically where you're sourcing news from.

Reply:
I would echo the point of the two other peer reviewers regarding the links. Ultimately, the point of the `[]` is to add descriptive text between the brackets that will allow the reader to identify what you are linking to. Knowing how to do this correctly will be useful for future coding assignments